installer/tue-install-impl.bash

```diff
@@ -841,14 +834,15 @@ if [ -n "$TUE_INSTALL_SYSTEMS" ]; then
 fi
```

Comment:
I think tue-install-error pipe must be added to all the functions in this PR itself, as it is an implication of all the changes made here.

Reply:
You are free to do so, I don't have the time for it now.